runtime/port/sysvipc/j9sharedhelper.c

```diff
@@ -597,6 +602,83 @@ ControlFileCloseAndUnLock(struct J9PortLibrary* portLibrary, intptr_t fd)
 	return J9SH_SUCCESS;
 }
 
+
+#if defined(J9OS_I5)
+/**
+ * @internal
+ * @brief This function validates if the current user has *ALLOBJ and *SECADM special authorities.
+ *
+ * This function is used to help manage files used to control System V objects (semaphores and memory)
+ * @return TRUE if current user has ALLOBJ and *SECADM special authorities.
+ */
+static BOOLEAN IsAllObjAndSecAdmUser()
```

Comment:
For consistency with the rest of the code base, can you modify this function to start with a lowercase letter?  `isAll..`